CHANGES.md

```diff
@@ -0,0 +1,7 @@
+# CHANGES
+
+## 0.1.0
+
+* First container release uses [cgpBigWig-0.5.0-rc1](https://github.com/cancerit/cgpBigWig/releases/tag/0.5.0-rc1)
```

Comment:
First release will need to use an actual release not rc, ensure corrected in the release branch

Reply:
Will change when making a release.